include/z64player.h

```diff
@@ -703,7 +703,7 @@ typedef struct WeaponInfo {
 
 #define LEDGE_DIST_MAX 399.96002f
 
-#define PLAYER_STATE1_0 (1 << 0)
+#define PLAYER_STATE1_LOADING (1 << 0) // Transitioning to new scene
```

Comment:
fwiw this is one that is small enough (in terms of uses) and has enough documentation surrounding it that it could probably be named now. I would agree with `PLAYER_STATE1_EXITING_SCENE`.